test/utils/shared.ts

```diff
@@ -1,5 +1,35 @@
 import { AzureCliCredentials } from "@azure/ms-rest-nodeauth";
 import crypto from "crypto";
+import { Frame } from "playwright";
+
+export enum AccountType {
+  Tables = "Tables",
+  Cassandra = "Cassandra",
+  Gremlin = "Gremlin",
+  Mongo = "Mongo",
+  Mongo32 = "Mongo32",
+  SQL = "SQL",
+}
+
+export const defaultAccounts: Record<AccountType, string> = {
+  [AccountType.Tables]: "portal-tables-runner",
+  [AccountType.Cassandra]: "portal-cassandra-runner",
+  [AccountType.Gremlin]: "portal-gremlin-runner",
+  [AccountType.Mongo]: "portal-mongo-runner",
+  [AccountType.Mongo32]: "portal-mongo32-runner",
+  [AccountType.SQL]: "portal-sql-runner-west-us",
+};
+
+const resourceGroup = process.env.DE_TEST_RESOURCE_GROUP ?? "runners";
+const subscriptionId = process.env.DE_TEST_SUBSCRIPTION_ID ?? "69e02f2d-f059-4409-9eac-97e8a276ae2c";
```

Comment:
Is it ok to hardcode sub id here?

Reply:
We already hardcode it in the webpack config (I didn't remove that, actually, but could now). It's still overridable via the new env vars (DE_TEST_ ...)